assets/sass/config/_variables-mui3.scss

```diff
@@ -124,7 +124,7 @@ $c-yellow-y-900: #1f1400;
 $c-yellow-y-1000: #000;
 $c-violet-v-0: #fff;
 $c-violet-v-10: #fbf9ff;
-$c-violet-v-50: #dbc5ff;
+$c-violet-v-50: #e3d1ff;
```

Comment:
This color token has a new value in Figma, and since this variable was not used anywhere so far, I updated it's value instead of creating a duplicate variable with different value